composer.json

```diff
@@ -43,7 +43,7 @@
     "require-dev": {
         "phpunit/phpunit": "~8.0|~7.0|~9.0",
```

Comment:
May want to allow phpunit 10 too

https://laravel.com/docs/10.x/upgrade#updating-dependencies

> Optionally, if you wish to use [PHPUnit 10](https://phpunit.de/announcements/phpunit-10.html), you should delete the processUncoveredFiles attribute from the <coverage> section of your application's phpunit.xml configuration file. Then, update the following dependencies in your application's composer.json file:

> nunomaduro/collision to ^7.0
> phpunit/phpunit to ^10.0

Reply:
I saw that, but even allowing PHPUnit 10 may not work for us unless we drop support for other versions right? 

It would be possible to install L9/L10+PHP unit 9, and those would need the `processUncoveredFiles` right ?